python/src/tests/de_result.json

```diff
@@ -0,0 +1 @@
NSG00000162191", "ENSG00000089280", "ENSG00000101608", "ENSG00000142168", "ENSG00000172543", "ENSG00000141293", "ENSG00000100100", "ENSG00000145287", "ENSG00000127152", "ENSG00000198899", "ENSG00000167815", "ENSG00000009790", "ENSG00000211751", "ENSG00000101745", "ENSG00000034510", "ENSG00000133872", "ENSG00000132424", "ENSG00000076928", "ENSG00000198668", "ENSG00000129824", "ENSG00000099622", "ENSG00000145247", "ENSG00000160613", "ENSG00000132507", "ENSG00000197540", "ENSG00000269893", "ENSG00000213658", "ENSG00000133321", "ENSG00000116824", "ENSG00000135486", "ENSG00000168421", "ENSG00000136732", "ENSG00000107742", "ENSG00000198918", "ENSG00000125743", "ENSG00000142546", "ENSG00000139193", "ENSG00000125534", "ENSG00000204592", "ENSG00000173272", "ENSG00000173762", "ENSG00000111371", "ENSG00000125354", "ENSG00000165502", "ENSG00000162894", "ENSG00000180096", "ENSG00000096384", "ENSG00000169567", "ENSG00000213626", "ENSG00000160654", "ENSG00000215845", "ENSG00000198821", "ENSG00000116251", "ENSG00000164543", "ENSG00000152082", "ENSG00000168685", "ENSG00000160932", "ENSG00000069493", "ENSG00000071082", "ENSG00000156976", "ENSG00000089220", "ENSG00000110651", "ENSG00000081059", "ENSG00000111716", "ENSG00000109475", "ENSG00000161970", "ENSG00000136717", "ENSG00000114391", "ENSG00000110848", "ENSG00000196683", "ENSG00000167286", "ENSG00000008517", "ENSG00000169442", "ENSG00000147168", "ENSG00000136490", "ENSG00000172183", "ENSG00000089009", "ENSG00000241343", "ENSG00000149806", "ENSG00000175463", "ENSG00000182866", "ENSG00000188042", "ENSG00000008988", "ENSG00000198851", "ENSG00000166441", "ENSG00000134954", "ENSG00000277734", "ENSG00000137818", "ENSG00000131469", "ENSG00000156482", "ENSG00000211772", "ENSG00000072818", "ENSG00000182899", "ENSG00000163682", "ENSG00000204628", "ENSG00000171863", "ENSG00000181163", "ENSG00000109971", "ENSG00000105373", "ENSG00000145425", "ENSG00000108107", "ENSG00000174444", "ENSG00000233927", "ENSG00000104964", "ENSG00000142676", "ENSG00000161016", "ENSG00000124614", "ENSG00000172809", "ENSG00000170889", "ENSG00000114942", "ENSG00000227507", "ENSG00000144713", "ENSG00000251562", "ENSG00000197958", "ENSG00000134419", "ENSG00000174748", "ENSG00000112306", "ENSG00000122026", "ENSG00000188846", "ENSG00000089157", "ENSG00000162244", "ENSG00000122406", "ENSG00000142937", "ENSG00000198034", "ENSG00000164587", "ENSG00000177600", "ENSG00000196405", "ENSG00000156508", "ENSG00000186468", "ENSG00000063177", "ENSG00000105640", "ENSG00000130255", "ENSG00000105193", "ENSG00000143947", "ENSG00000145592", "ENSG00000115268", "ENSG00000118181", "ENSG00000142541", "ENSG00000171858", "ENSG00000136942", "ENSG00000148303", "ENSG00000167526", "ENSG00000108298", "ENSG00000198755", "ENSG00000229117", "ENSG00000198242", "ENSG00000137154", "ENSG00000083845", "ENSG00000140988", "ENSG00000147403", "ENSG00000197756", "ENSG00000231500", "ENSG00000213741", "ENSG00000149273", "ENSG00000100316", "ENSG00000177954", "ENSG00000168028", "ENSG00000105372"]}
```

Comment:
do we need this file in github?

Reply:
Yes, it is for mocking the R worker's DE result.